src/BinaryParsers/VersionConstants.cs

```diff
@@ -1,6 +1,5 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT         
```

Comment:
this file the change was automatic by build and test, I did not make any change.